src/app/components/Picture/index.tsx

```diff
@@ -0,0 +1,49 @@
+import classNames from 'classnames';
+import React, { useMemo } from 'react';
+import styles from './index.module.scss';
+import { Breakpoint, BreakpointWidths } from 'types/tailwind';
+
+type Sources = {
+  src: string;
+  media: Breakpoint;
+};
+
+type PictureProps = {
+  src: string;
+  srcSet?: Sources[];
+  title?: string;
+  className?: string;
+};
+
+export const Picture: React.FC<PictureProps> = ({
+  src,
+  srcSet,
+  title = 'image',
+  className,
+}) => {
+  const renderSources = useMemo(() => {
+    if (!srcSet) {
+      return <source srcSet={src} />;
```

Comment:
This is redundant, if srcSet is not set we can drop the source elements
```suggestion
      return null;
```

Reply:
fixed